base_phone/models/phone_common.py

```diff
@@ -5,11 +5,15 @@
 from openerp import models, api
 from openerp.addons.base_phone.fields import Phone
```

Comment:
The issue is actually in `File "/usr/lib/python2.7/site-packages/openerp/addons/base_phone/init.py", line 3, in `